stan/math/prim/mat/fun/to_matrix.hpp

```diff
@@ -1,14 +1,19 @@
 #ifndef STAN_MATH_PRIM_MAT_FUN_TO_MATRIX_HPP
 #define STAN_MATH_PRIM_MAT_FUN_TO_MATRIX_HPP
 
+#include <boost/math/tools/promotion.hpp>
+#include <stan/math/prim/scal/err/check_size_match.hpp>
+#include <stan/math/prim/scal/err/invalid_argument.hpp>
 #include <stan/math/prim/mat/fun/Eigen.hpp>
+#include <Eigen/Dense>
 #include <vector>
 
 namespace stan {
   namespace math {
     /**
      * Returns a matrix with dynamic dimensions constructed from
-     * an Eigen matrix which is either a row vector, column vector, or matrix.
+     * an Eigen matrix which is either
+     * a row vector, column vector, or matrix.
```

Comment:
This sentence cannot fit on 2 lines (of 80 characters), but I changed it now to:
     * Returns a matrix with dynamic dimensions constructed from an
     * Eigen matrix which is either a row vector, column vector,
     * or matrix.